packages/client/src/components/Wizard/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import ProgressMobileStepper from './ProgressBar.js'
+import WizardPage from './WizardPage.js'
+
+export default function Wizard(props) {
+
+  const [activeStep, setActiveStep] = React.useState(0);
+
+  const handleNext = () => {
+    setActiveStep((prevActiveStep) => prevActiveStep + 1);
+  };
+
+  const handleBack = () => {
+    setActiveStep((prevActiveStep) => prevActiveStep - 1);
+  };
+
+  return (
+    <div>
+          {props.content.map((page) => {
```

Comment:
what if instead of mapping over all the pages, we just did 

```jsx
<WizardPage>{props.content[activeStep] || someDefaultValue}</WizardPage> 
```

Then `WizardPage` can be simpler, or maybe we don't need it and can just use div for now

Reply:
Ah yeah you're right this would be cleaner -- I'll implement this